deploy/chart/Chart.yaml

```diff
@@ -2,5 +2,7 @@ apiVersion: v2
 name: ccm-linode
 description: The Linode Cloud Controller Manager (CCM) provides a way for Kubernetes clusters to access additional Linode services. Linode's CCM will automatically provision a Linode NodeBalancer for Kubernetes Services of type "LoadBalancer".
 type: application
-version: 1.0.0
+version: 0.0.0
```

Comment:
is this intentional?

Reply:
This is intentional, it is used by the `sed` in the helm release workflow to update the version dynamically based on the tag